src/renderer/screens/accounts/AccountList/GridBody.js

```diff
@@ -46,17 +47,7 @@ const GridBody = ({
       />
     ))}
     {showNewAccount ? <AccountCardPlaceholder key="placeholder" /> : null}
```

Comment:
as said IRL, i highly suspect it's important this is placed in the DOM between the visible and hidden